velodyne_pointcloud/tests/CMakeLists.txt

```diff
@@ -1,52 +1,53 @@
-### Unit tests
-#
-#   Only configured when CATKIN_ENABLE_TESTING is true.
-
 # these dependencies are only needed for unit testing
-find_package(roslaunch REQUIRED)
-find_package(rostest REQUIRED)
-find_package(tf2_ros REQUIRED)
+find_package(ament_cmake_gtest REQUIRED)
+
+ament_add_gtest(test_row_step test_row_step.cpp)
+ament_target_dependencies(test_row_step
+  rclcpp
```

Comment:
Your test isn't using `rclcpp` so this can be removed.

Reply:
Ah yes, but `datacontainerbase.h` *is* using rclcpp (for time), and since this test is using it in header-only mode, we still need the dependency.